static/sass/_pattern_cube-animation.scss

```diff
@@ -0,0 +1,70 @@
+$size: (
```

Comment:
Can you wrap these styles in a mixin similar to all the other patterns? For example: https://github.com/canonical-web-and-design/ubuntu.com/blob/master/static/sass/_pattern_gaming.scss